src/include/catalog/postgres/name_builder.h

```diff
@@ -3,29 +3,36 @@
 #include <string>
 
 #include "catalog/postgres/pg_defs.h"
+#include "common/exception.h"
 
 namespace terrier::catalog::postgres {
-/* NameBuilder generates names for database objects such as constraints and indexes
+/**
+ * NameBuilder generates names for database objects such as constraints and indexes
  *
  * Following Postgres naming convention
- * */
+ */
 class NameBuilder {
  public:
-  /*
+  /**
+   *  FK: foreign key, KEY: UNIQUE, EMPTY: empty input
+   */
+  enum Types { FK, KEY, EMPTY };
```

Comment:
Rename `Types` to `ObjectType`.
Rename `FK` to `FOREIGN_KEY`
Rename `KEY` to `UNIQUE_KEY`
Rename `EMPTY` to `NONE`